waypoint/routes/sse.py

```diff
@@ -0,0 +1,133 @@
+import asyncio
```

Comment:
Minor naming consistency note: the folder `routes` should be `routers`. Just the convention in other services. Also for tests/routes